src/components/classifier/MultiAnswerClassifier.js

```diff
@@ -270,69 +296,69 @@ MultiAnswerClassifier.propTypes = {
     subjectsSeenThisSession: PropTypes.array,
     task: PropTypes.object,
     answers: PropTypes.array,
-    classifierActions: PropTypes.any
+    classifierActions: PropTypes.any,
+    inMuseumMode: PropTypes.bool,
```

Comment:
Doesn't look like this prop is used by this component.

Reply:
Good catch! It isn't. Will remove :)